rest-service/manager_rest/test/mock_blueprint/host_agent_plugin.yaml

```diff
@@ -0,0 +1,6 @@
+plugins:
+  cool:
+    executor: host_agent
+    source: https://github.com/cloudify-cosmo/cloudify-script-plugin/archive/1.2.zip
```

Comment:
but this is going to be actually installed in the test? unfortunate... making the tests even slower :/

Reply:
Suggestions?

Reply:
who knows. Have a plugin locally i guess. And this of course applies to all tests that download plugins (note that plugins are the slowest part of the tests currently)